packages/annotorious/package.json

```diff
@@ -48,7 +48,7 @@
     "vitest": "^1.4.0"
   },
   "dependencies": {
-    "@annotorious/core": "*",
+    "@annotorious/core": "3.0.0-rc.19",
```

Comment:
*Can only be bumped to the `^3.0.0-rc.21` after the respective version will get published on NPM